### PR TITLE
Update doc and changelog about .NET4 requirement

### PR DESCRIPTION
**Pull request by judubu**
## Purpose of this PR

Doc and changelog update to warn users about the .NET4 requirement in Editor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,14 +95,18 @@ Timeline playback is multi-threaded using the C# Job System.
 Access to variant selection, model details, and payload state are all accessible via
 the inspector on the game object at which these features were authored.
 
-# Unity 2017 Users
+## Exporting USD files
```
Comment (mfe) [inline README.md:98]: I'm going to refactor this a little bit.

**Comment by mfe** [comment]
@cguer @judubu did an iteration on the export doc.
Feel free to fix. The idea was to better explain why use an option more than another.

File: README.md [hunk 2]
```diff
@@ -31,10 +31,9 @@ settings under Edit > Project Settings > Quality:
 ## Requirements
 
 * Windows / OSX
-* Unity version: 2018.3 (2018.3.4f1 and up) and 2019.1 (2019.1.0b2 and up)
-See ["2017 users" section](#unity-2017-users) if you want to use the plugin in Unity 2017.4
-* To build a standalone, Api Compatibility Level needs to be set to .NET 4.x
-   In Edit > Project Settings > Player :
+* Unity version: 2019.4 and up
+* Api profile .NET 4.x is no longer required but still provides better performances than .NET 2
```
Comment (cguer) [inline README.md]: Is `profile` the correct word here? Maybe we could say `Api Compatibility Level` to mirror what is in the interface?

File: README.md [hunk 3]
```diff
@@ -101,14 +95,39 @@ Timeline playback is multi-threaded using the C# Job System.
...
+
+>  **Note:** This feature has no dependency to and is not based on the Recorder package.
+
+### Exporting via USD menu (Legacy)
+
+A last option to export a USD Composition is to use the legacy export via the Editor main menu :
+* Select **USD > Export Selected**.
 
-The official package is not compatible with 2017.4 but a separate branch is maintained to allow 2017 users to benefit from the last developments.
+Note that this option is not recommended as it offers very few export options.
```
Comment (cguer) [inline README.md]: To be technically correct, I would change the end of the sentence to something like `as it doesn't offer any export options`.

**Review by cguer — APPROVED** [review]
Minor suggestions/questions but otherwise looks good to me 👍!

File: README.md (hunk 2 above)
Comment (judubu) [reply README.md]: Yeah profile is the term used in the doc, but I'll change that

File: README.md (hunk 3 above)
Comment (judubu) [reply README.md]: 👍

Comment (judubu) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (judubu) [reply README.md]: Done

**Pull request merged**